assets/styles/variables/_specials.scss

```diff
@@ -315,6 +315,15 @@ $pullquote-margin-top: 1em !default;
 /// @type String
```

Comment:
This parameter (and the bottom margin) should be `@type String | Map` for added flexibility. No need to change the default value.